raiden/api/rest.py

```diff
@@ -240,6 +242,10 @@
         '/light_clients/',
         LightClientResource
     ),
+    (
+        '/secrets_light/',
```

Comment:
This is a payment related feature, probably is better if you use ```payments_light/register_secret``` or something like that.

Reply:
I don't like `payments_light/register_secret`, It feels more like an RPC, rather than a REST API. I thought it as we were someway adding a new 'secret'. What do you think about:
-  `payments_light/{payment_id}/secrets` moving payment_id from  the body to the path, to make it restful
or
-  `payments_light/secrets` keeping payment_id in the body of the request?

I personally prefer the first one.

Reply:
Both are good, i think any option is good if it's under the payments_light path since a related payments feature

Reply:
`payments_light/secrets`  looks beter to me, but i will name it `payments_light/register_onchain_secret`